vivisect/qt/main.py

```diff
@@ -115,6 +116,8 @@ def __init__(self, vw):
         self.addHotKeyTarget('file:connecttoserver', self._menuShareConnectServer)
         self.addHotKey('ctrl+w', 'file:quit')
         self.addHotKeyTarget('file:quit', self.close)
+        self.addHotKey('ctrl+m', 'view:memory')
+        self.addHotKey('ctrl+f', 'view:funcgraph')
```

Comment:
`ctrl+f` I would expect to be a find function, not a new funcgraph.

Reply:
```suggestion
        self.addHotKey('ctrl+g', 'view:funcgraph')
```